keda/values.yaml

```diff
@@ -119,4 +119,16 @@ priorityClassName: ""
 ## Extra environment variables that will be passed onto Keda operator and metrics api service
 env:
 # - name: ENV_NAME
-#   value: 'ENV-VALUE'
\ No newline at end of file
+#   value: 'ENV-VALUE'
+
+prometheus:
```

Comment:
Do we turn them on or off by default?

Reply:
Good catch. That was an oversight on my end - they were turned on for testing purposes.